tests/test_system_models.py

```
@@ -0,0 +1,133 @@
+    """Test SystemHealth model initialization and validation."""
+    health = SystemHealth(**VALID_SYSTEM_HEALTH)
+    assert health.subsystem == "network"
+    assert health.status == "ok"
+    assert health.status_code == 0
+    assert health.status_message == "System healthy"
+    assert health.last_check == 1000
+    assert health.next_check == 2000
+
+
+@pytest.mark.parametrize(
+    "invalid_data,expected_error",
```

Ruff (0.8.2)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)